modules/backup-filesystem/snapshot.go

```diff
@@ -40,6 +41,8 @@ func (m *BackupFileSystemModule) GetObject(ctx context.Context, backupID, key st
 		return nil, backup.NewErrInternal(errors.Wrapf(err, "get object '%s'", metaPath))
 	}
 
+	monitoring.GetMetrics().BackupRestoreDataTransferred.WithLabelValues(m.Name(), "class").Add(float64(len(contents)))
```

Comment:
WithLabelValues might panic and we don't want to panic because of a metric.
You could use GetMetricWithLabelValues instead

Also might not be a good place to put metrics here since getObject and other methods are class agnostic.